specification/botservice/resource-manager/Microsoft.BotService/preview/2023-09-15-preview/networksecurityperimeter.json

```diff
@@ -0,0 +1,309 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-09-15-preview",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "NetworkSecurityPerimeterConfigurationList": {
+      "description": "Result of the List NetworkSecurityPerimeterConfiguration operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of Network Security Perimeter configurations",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          }
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfiguration": {
+      "description": "Network Security Perimeter configuration",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "id": {
+          "description": "Fully qualified identifier of the resource",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the resource",
+          "type": "string"
+        },
```

Comment:
can you please use the proxy resource reference from commontypes ?

Reply:
Sure.

Reply:
PLease use v5 not v1

Reply:
Why changed it back ?

Reply:
why changed it back w/o using common type?